apps/web/__tests__/support/pageObjects/LoginPageObject.ts

```diff
@@ -0,0 +1,24 @@
+import {paths} from '../../../utils/paths'
+export class LoginPageObject {
+  forgotPassword() {
+    cy.getByTestId('login-page-reset-button').click();
+    cy.url().should('contain', '/reset-password');
+  }
+
+  successLogin() {
+    cy.getFixture('account').then((fixture) => {
+      cy.get(`[type="email"]`).type(fixture.email, { delay: 0 });
+      cy.get(`[type="password"]`).type(fixture.password, { delay: 0 });
+    });
+    cy.getByTestId('checkbox').realHover().click();
+    cy.get(`[type="submit"]`).click();
+    cy.contains('Sneaker hot drops').should('exist');
+  }
+
+  navigation() {
+    cy.getByTestId('login-page-signup-button').click();
+    cy.url().should('contain', paths.authSignup);
+    cy.getByTestId('signup-page-login-button').click();
+    cy.url().should('contain', paths.authLogin);
+  }
```

Comment:
Suggestion:
```suggestion
  clickSignUpLink() {
    cy.getByTestId('login-page-signup-button').click();
    return this;
  }

  clickLoginLink() {
    cy.getByTestId('signup-page-login-button').click();
    return this;
  }
  
  isURL(path: string) {
    cy.url().should('contain', path);
    return this;
  }
```

and usage:

```ts
login
  .clickSignUpLink()
  .isURL(paths.authSignup)
  .clickLoginLink()
  .isURL(authLogin);
```

Reply:
done @FRSgit